spp_farmer_registry_base/models/farmer.py

```diff
@@ -9,7 +9,25 @@ class Farmer(models.Model):
     farmer_national_id = fields.Char(string="National ID Number")
     farmer_household_size = fields.Integer(string="Farmer Household Size")
     farmer_postal_address = fields.Char("Postal Address")
-
+    marital_status = fields.Selection(
+        [
+            ("single", "Single"),
+            ("married_monogamous", "Married Monogamous"),
+            ("married_polygamous", "Married Polygamous"),
```

Comment:
We cannot have this as default. `Married Polygamous` is not legal in every country

Reply:
Moved